packages/perspective/src/js/perspective.parallel.js

```diff
@@ -185,6 +185,8 @@ class WebSocketWorker extends worker {
 export default {
     worker: function (url) {
         if (window.location.href.indexOf(__SCRIPT_PATH__.host()) === -1 && detectIE()) {
+            // TODO: Should this be here? Where is it defined?
+            // eslint-disable-next-line no-undef
             return perspective;

```

Comment:
Where does `perspective` come from in this branch of code? Seems to be completely undefined in IE if I read this correctly.

Reply:
This is a nasty edge case, specifically regarding when IE tries to load perspective as a WebWorker, but as a cross-origin request.  This is not supported by IE at all as far as I can tell, so we instead download the inline version of the asmjs build of perspective, by [inlining the script tag into `<head>` via Javascript](https://github.com/jpmorganchase/perspective/blob/master/packages/perspective/src/js/perspective.parallel.js#L25).  This binds the `perspective` symbol globally, and a reference to this symbol is returned here when an attempt to instantiate a worker is made.

Reply:
Ah okay, that makes sense. In that case it should be enough to change this to `window.perspective` to indicate that we're using a global variable. I'll update the PR with that change.